crates/swift-bridge-ir/src/codegen/generate_swift/opaque_copy_type.rs

```diff
@@ -112,11 +112,36 @@ extension {ffi_repr_name}: SwiftBridgeGenericCopyTypeFfiRepr {{}}"#,
         )
     };
 
+    let ext_equatable = if ty.attributes.equatable {
```

Comment:
Please add a codegen test.
One benefit is that since our codegen tests run much more quickly than the integration tests they allow us to more easily iterate/fix/change the codegen as we work.

Example: https://github.com/chinedufn/swift-bridge/blob/c15279c5a412fbc597e6831673b6bf850ddb4217/crates/swift-bridge-ir/src/codegen/codegen_tests/opaque_rust_type.rs#L148-L209